README.md

```diff
@@ -9,7 +9,7 @@ Polyfill for the `headingoffset` attribute. Related discussion in https://github
 Add the polyfill:
 
 ```html
-<script type="module" src="headingoffset-polyfill.js"></script>
+<script src="headingoffset-polyfill.js"></script>
```

Comment:
I'm not sure what would your usage recommendation be? Do you have one?  If you include it in the head in blocking fashion like this it will have to fully download and execute and therefore the handlers will be there - but they'll be pretty busy if it is a big page downloading then.  You could also do it like this at the end but then you have to do a full document scan and if it is a big document there could be some time where the headings are wrong.  Maybe an async attribute is a nice compromise? Maybe just add a note to the effect that the script can be used in blocking or non-blocking fashion?

Reply:
> …what would your usage recommendation be? Do you have one?

No, I don’t have a strong opinion on where folks should put the script tag (and/or whether they should use `defer`, `async`, etc.)

> Maybe just add a note to the effect that the script can be used in blocking or non-blocking fashion?

Sure, I’m open to adding a note outlining options and trade-offs. Is there a note like this in another polyfill’s README that I could read over for wording/formatting inspiration?

Reply:
No, I don't have any examples I can think of - but I always wonder about this exact thing when I find a polyfill. Maybe just "The script will scan the existing document and monitor for changes after that" and people can figure the rest out for themselves :). I wouldn't block anything based on this comment btw... I would use async probably on my site :)

Reply:
Thanks! Sounds good to me; I added that note via https://github.com/smockle/headingoffset-polyfill/pull/16/commits/32887ec19affac8886427ee86910c6cdd3055924.